### ISSUE TITLE
E-mail change confirmation send to wrong(?) mailbox

### ISSUE THREAD

**Issue by rvbrakel** [svthalia/concrexit#1144]
### Describe the bug
After changing your e-mail address the last confirmation is send to the **new** inbox, while it is stated after clicking on the confirmation URL it was send to the **old** mailbox. 

### How to reproduce
Steps to reproduce the behaviour:
1. Change your mailadres
2. Confirm request via mail in your **old** mailbox
3. Confirm request via mail in your **new** mailbox
4. See on Thalia webpage that confirmation is send to **old** mailbox, but you will receive mail in **new** inbox.

### Expected behaviour
See my explanation above, is either easy fix in html changing "old" to "new" or easy fix to send mail to **old** e-mail address

### Screenshots
![afbeelding](https://user-images.githubusercontent.com/45627978/83969799-46a46980-a8d2-11ea-91b3-c37861904171.png)

**Comment by se-bastiaan** [comment]
I'm not sure I get the issue exactly. I have everything and what should happen is:

1. Change email
2. Accept confirmation in both the old and new inboxes
3. Based on which one you open you get a different text

**Comment by se-bastiaan** [comment]
I did try this and the only problem I found is that the last confirmation email that you get is in English.

![Screenshot 2020-08-31 at 21 11 26](https://user-images.githubusercontent.com/1799914/91757635-ef5f1e00-ebce-11ea-92ec-3e3756717bcc.png)



And I think I now get the confusion. The two pages point to each other but if you accept the one from your new inbox last then you still get the third and final confirmation in your new inbox.
